test/e2e/Core_populateFromArray.spec.js

```diff
@@ -91,6 +91,41 @@ describe('Core_populateFromArray', () => {
     expect(output).toEqual([[1, 3, 12, 'test'], [1, 4, 13, 'test']]);
   });
 
+  it('should populate value for array data when array selection is changed to empty', () => {
+    // Resolving issue #5675: https://github.com/handsontable/handsontable/issues/5675
+    let output = null;
+    const dataArray = arrayOfArrays();
+    dataArray[0][0] = ['2011'];
+
+    handsontable({
+      data: dataArray,
+      afterChange(changes) {
+        output = changes;
+      }
+    });
+    populateFromArray(0, 0, [[[]]], 0, 0);
+
+    expect(output).toEqual([[0, 0, ['2011'], []]]);
+  });
+
+  it('should populate value for array data when bound data begins as empty with new row', () => {
+    // Resolving issue #5675: https://github.com/handsontable/handsontable/issues/5675
+    let output = null;
+    const dataArray = []; // arrayOfArrays();
+    // dataArray[0][0] = ['2011'];
```

Comment:
Is this line needed? Also, the `// arrayOfArrays();` the line above seems to be a leftover from the previous test case.

Reply:
Indeed. These were just junk comments that I had added as I modified another copied test case. I've removed them now.